src/audits/accessibility/aria-valid-attr.js

```diff
@@ -43,6 +43,15 @@ class ARIAValidAttr extends Audit {
   }
 
   /**
+   * @return {!Array<string>}
```

Comment:
done. not quite as compact...we require line breaks after curly brackets. Compromise! :D